src/coreclr/vm/codeversion.h

```diff
@@ -61,6 +61,7 @@ class NativeCodeVersion
     NativeCodeVersionId GetVersionId() const;
     BOOL IsDefaultVersion() const;
     PCODE GetNativeCode() const;
+    PTR_PCODE GetNativeCodeSlot() const;
```

Comment:
This isn't used as-is, so can be removed. It's there because I initially tried to reuse the code slot from the code version in the async resumption stub, but I was not able to make it work reliably. Instead `getAsyncResumptionStub` just allocates a new slot and later (after jitting) writes the pointer into it.